src/config.rs

```diff
@@ -763,15 +763,26 @@ pub struct Plugins {
     pub prewarmer: Option<Prewarmer>,
 }
 
+pub trait Plugin {
+    fn is_enabled(&self) -> bool;
+}
+
 impl std::fmt::Display for Plugins {
     fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        fn is_enabled<T: Plugin>(arg: Option<&T>) -> bool {
```

Comment:
Cleaner and safer to write it this way:

```
if let Some(ref arg) = arg {
    arg.is_enabled()
} else {
    false
}```

Reply:
That's a valid suggestion. Thanks. Done.